crontab

```diff
@@ -14,7 +14,7 @@ YANGCATALOG_CONFIG_PATH=<YANGCATALOG_CONFIG_PATH>
 5 5 * * * (cd ~ ; source bin/activate ; echo "`date` starting reviseTreeType" >> /var/yang/logs/crons-log.log ; cd utility ; python reviseTreeType.py)
 5 6 * * * (cd ~ ; source bin/activate; echo "`date` starting openconfigPullLocal" >> /var/yang/logs/crons-log.log ; cd ietfYangDraftPull ; python openconfigPullLocal.py )
 5 12 * * 0 (cd ~ ; source bin/activate ; echo "`date` starting reviseSemver" >> /var/yang/logs/crons-log.log ; cd parseAndPopulate ; python reviseSemver.py)
-5 10 * * * (cd ~ ; source bin/activate ; echo "`date` starting userReminder" >> /var/yang/logs/crons-log.log ; cd utility ; python userReminder.py)
+5 10 * 3,9 1 (cd ~ ; source bin/activate ; echo "`date` starting userReminder" >> /var/yang/logs/crons-log.log ; cd utility ; python userReminder.py)
```

Comment:
Wouldn't it be better if `5 10 * * 1` will be used?

Reply:
The script used to check if it's March or September, but we could have it run every month if that would be preferred. I also made a mistake, the fifth field selects the day of the week, so this would run every Monday, it should be `5 10 1 * *` if we want it to run on the first day of every month.

Reply:
I would say that once a week (or once a month) is ideal. Every six months seems like a very long time.

Reply:
Ok, I'll change it to once a month, once a weeks seems excessive given that an email is also sent whenever a new user is registered as well.